src/main/java/com/brightpath/learnify/domain/auth/PermissionAccessService.java

```diff
@@ -192,4 +193,10 @@ public void deletePermissionToResourceForUser(UUID resourceId, String userId) {
     private String permissionAccessId(UUID resourceId, ResourceType resourceType) {
         return resourceType.toString() + ":" + resourceId.toString();
     }
+
+    @Transactional
+    public PermissionLevel editResourcePermissionModel(UUID resourceId, ResourceType resourceType, PermissionLevel permissionLevel) {
+        permissionAccessRepository.editResourcePermissionModel(permissionAccessId(resourceId, resourceType), permissionLevel);
```

Comment:
shouldn't we check what will be return bu this edit?

Reply:
Maybe we should🤔
I will think what is the best option here